packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/internal/SyncConfigurationImpl.kt

```diff
@@ -53,11 +52,12 @@ import kotlinx.coroutines.TimeoutCancellationException
 import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.withContext
 import kotlinx.coroutines.withTimeout
+import org.mongodb.kbson.BsonValue
 
 @Suppress("LongParameterList")
 internal class SyncConfigurationImpl(
     private val configuration: InternalConfiguration,
-    internal val partitionValue: PartitionValue?,
+    internal val partitionValue: BsonValue?,
```

Comment:
I wasn't aware, I removed it because the `PartitionValue` had some implementation details on how to serialize `BsonValues`.. so I see an opportunity to remove duplicate code :)